packages/react-core/src/components/Wizard/__tests__/Wizard.test.tsx

```diff
@@ -205,3 +205,50 @@ describe('Wizard', () => {
     expect(main).toHaveAttribute('aria-labelledby', 'main-aria-labelledby');
   });
 });
+
+  test.only('wiz with disabled steps', () => {
+    const steps: WizardStep[] = [
+      { name: 'A', component: <p>Step 1</p>},
+      { name: 'B', component: <p>Step 2</p>, isDisabled: true },
+      { name: 'C', component: <p>Step 3</p>},
+      { name: 'E', component: <p>Step 4</p>, isDisabled: true },
+      { name: 'G', component: <p>Step 5</p> },
+
+    ];
+    
+    render(
+      <Wizard
+        navAriaLabel="nav aria-label"
+        description="This wizard step is disabled"
+        steps={steps}
+      />
+    );
+    const button1Disabled = screen.getByRole('button',{ name: "B" });
+    const button2Disabled = screen.getByRole('button',{ name: "E" });
+
+    expect(button1Disabled).toHaveAttribute('disabled');
+    expect(button2Disabled).toHaveAttribute('disabled');
+  });
+
+  test.only('wiz with disable sub step', () => {
+    const steps: WizardStep[] =   [
+      {
+        name: 'First step',
+        steps: [
+          { name: 'Substep A', component: <p>Substep A content</p> },
+          { name: 'Substep B', component: <p>Substep B content</p>,  isDisabled: true }
+        ]
+      },
+    ];
+    
+    render(
+      <Wizard
+        navAriaLabel="nav aria-label"
+        description="This wizard step is disabled"
+        steps={steps}
+      />
+    );
+    const buttonDisabled = screen.getByRole('button',{ name: "Substep B" });
+
+    expect(buttonDisabled).toHaveAttribute('disabled');
+  });
```

Comment:
I think it would be beneficial to also test `next` and `back` behavior to ensure that disabled steps are skipped over as we expect them to be. Maybe also a test which verifies that clicking on the name of a disabled step doesn't bring its content up.